com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/CopyCustomResourcesBuilder.java

```diff
@@ -41,8 +41,10 @@
                 this.project.findResourcePaths(s, paths);
                 for (String path : paths) {
                     IResource r = this.project.getResource(path);
-                    b.addInput(r);
-                    b.addOutput(r.output());
+                    if (r.isFile()) {
```

Comment:
To be honest, I think a missing file is a build error?
I'd be very wary spreading this type of behavior.

I'd suggest that you use a setup script to change/swap the game.project for that particular user.

Reply:
About our use case: This isn't about changing game.project at build time. It's about quickly adding gitignored settings that only affect the developer currently editing the game, like setting the current startup scene, making the window small and whatnot. game.project is versioned, so any edit to it would still risk being commited.

Reply:
A workaround to this is adding `env.lua` to a directory and adding that subdirectory to `custom_resources`. That way bob won't complain about the missing file since it just crawls the dir and adds every file in there.

Reply:
I don't think it's very harmful to not bundle custom_resources that are anyway missing, but I get your point as well.

Anyway, if you decide to make this an error, it should be consistent between the editor and bob.

Reply:
In your use case, are you specifying a _folder_ or a _specific file_ in the `custom_resources` setting?

Reply:
A specific file. If I specify a folder, that folder may or may not contain `env.lua`, which is fine and it works, but it's adding a bit of friction for the user by hinding `env.lua` away in a subdirectory.

Reply:
The way we're leaning towards currently is that if you've specified a specific file, it will generate an error.
If you specified a directory, it will only yield a warning.

Reply:
That's fair, but how do you know if the thing that's missing is a directory or a file?

Reply:
Very good point!

Reply:
In light of that, we're more inclined to always make is a build error, I'm afraid.

Reply:
No worries. As I said before, I already worked around it. Want me to update the PR to cause a build error in the editor? I won't be able to do it now, but can get to it sometime next week.

Reply:
Please do, when/if you have the time!

Reply:
I'm moving this one to 1.2.171. Is the only thing left to do to generate a build error?

Reply:
Yup. I didn't give up on this, it's just that I had a rollercoaster of a last month.